README.md

```diff
@@ -139,20 +161,23 @@ Response
 
 ```json5
 {
-  "some file id": "some token",
-  "another file id": "some other token"
+  "sessionId": "mySessionId",
+  "files": {
+    "someFileId": "someFileToken",
+    "someOtherFileId": "someOtherFileToken"
+  }
 }
 ```
 
-### 3.2 Send File
+### 4.2 Send File
 
 The file transfer.
 
-Use the `fileId` and its file-specific `token` from `/send-request`.
+Use the `sessionId`, the `fileId` and its file-specific `token` from `/send-request`.
 
 This route can be called in parallel.
 
-`POST /api/localsend/v1/send?fileId=some file id&token=some token`
+`POST /api/localsend/v2/send?sessionId=mySessionId&fileId=someFileId&token=someFileToken`
```

Comment:
`sessionId` field why not add to header?

```http
POST /api/localsend/v1/send?fileId=some file id&token=some token

SessionID: <uuid>
```

Reply:
What advantages do it have? I want to keep the API as simple as possible.

Reply:
send-request via set-cookie header set cookie to client cookiejar

only need client-side supports cookie manages

Reply:
session management is cookie design usage

Reply:
There is no need to follow the convention as we develop a REST API which does not run on any browser. Having `sessionId` as request parameter is explicit and so we can have multiple sessions between two peers.